presentation-compiler/src/main/dotty/tools/pc/completions/ScalaCliCompletions.scala

```diff
@@ -26,7 +26,7 @@ class ScalaCliCompletions(
 
   def contribute(dependency: String) =
     val completions = coursierComplete.complete(dependency)
-    val (editStart, editEnd) = CoursierComplete.inferEditRange(pos.point, text)
+    val (editStart, editEnd) = CoursierComplete.inferEditRange(pos.end, text)
```

Comment:
Why this change?

Reply:
point in synthetic span (and when you modify it it becomes synthetic), points to the start, not end